apps/src/assetManagement/animationLibraryApi.js

```diff
@@ -46,6 +49,58 @@ export function updateDefaultList(listData) {
     });
 }
 
+/* Regenerates the animation JSON list of the sprites in SpriteLab
+ */
+export function regenerateDefaultJSON() {
+  getDefaultList().then(json => {
+    let orderedKeys = [];
+    let propsByKey = {};
+    let parsed = spriteManifest;
+    let animations = parsed['metadata'];
+    for (let sprite of json.default_sprites) {
+      let animation_metadata = animations[sprite.key];
+      let props = {};
+      props['name'] = sprite.name;
+      props['sourceUrl'] = `https://studio.code.org${
+        animation_metadata['sourceUrl']
+      }`;
+      props['frameSize'] = animation_metadata['frameSize'];
+      props['frameCount'] = animation_metadata['frameCount'];
+      props['looping'] = animation_metadata['looping'];
+      props['frameDelay'] = animation_metadata['frameDelay'];
+      props['version'] = animation_metadata['version'];
+      props['categories'] = animation_metadata['categories'];
```

Comment:
nit: i think we can make this more readable by using [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) and the [object initializer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer) notation:

```javascript
const { sourceUrl, frameSize } = animations[sprite.key];
const props = {
  name: sprite.name,
  sourceUrl: `https://studio.code.org${sourceUrl}`,
  frameSize
};
```